plugins/scaffolder-node/src/actions/executeShellCommand.ts

```diff
@@ -29,8 +30,10 @@ export type ExecuteShellCommandOptions = {
   args: string[];
   /** options to pass to spawn */
   options?: SpawnOptionsWithoutStdio;
-  /** stream to capture stdout and stderr output */
```

Comment:
I think that we will need to bring this back to deprecate it properly, and will need to add the `@deprecated` tsdoc to it to get it to show up in the `api-reports` and also for consumers.

Reply:
```ts
/**
* @deprecated - please provide a logger instead.
* stream to capture stdout and stderr output
*/
logStream?: Writeable;
```